modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -129,6 +135,61 @@ public void testConfigurationArrayValues() throws Exception {
 			(String[])properties.get("configUntypedStringArray"));
 	}
 
+	@Test
+	public void testConfigurationDeprecatedFileExtension() throws Exception {
+		String configurationPid = _CONFIGURATION_PID_PREFIX.concat(
+			".testConfigurationDeprecatedFileExtension");
+
```

Comment:
Use separate PIDs so that you can confirm that the cfg extension actually doesn't create a configuration